package.nls.json

```
@@ -462,6 +462,7 @@
     "AWS.samcli.initWizard.schemas.failed_to_load_resources": "Error loading schemas in registry {0}.",
     "AWS.samcli.initWizard.source.error.notFound": "Project created successfully, but main source code file not found: {0}",
     "AWS.samcli.initWizard.source.error.notInWorkspace": "Could not open file '{0}'. If this file exists on disk, try adding it to your workspace.",
+    "AWS.samcli.initWizard.completionMessage": "SAM application {0} created successfully. Debug configuration(s) {1} added to `launch.json`.",
```

Space and layout are important here (even more than usual) since the message has a lot of info and we want the user to consume it.

- We can save space by removing noise-words: "successfully" is always redundant, messages only need to mention what they did, not that it was a "success".
- The application name should at least have quotes, otherwise it blends into the middle of the sentence.
- The variable part should always go at the end where possible. Here, the debug configurations are a big list of variable values. 

```suggestion
    "AWS.samcli.initWizard.completionMessage": "Created SAM application \"{0}\" and added launch configurations to `launch.json`: {1}",
```

If VSCode renders bulleted lists in messages then we could make the message even more readable, but I assume that isn't possible.

---

Looks like showInformationMessage et al doesn't support MarkdownStrings, unfortunately. I don't think bullet points are supported but newlines should be, would that be acceptable (possibly with manually putting a bullet point character/asterisk in)?

---

Correction: line breaks (at least, `\n`) do not work in notifications (they do work in modals though).

---

newlines would be great but otherwise just the suggestion in my comment above is the best we can do